pkg/block/fetcher.go

```diff
@@ -184,6 +184,12 @@ func NewBaseFetcher(logger log.Logger, concurrency int, bkt objstore.Instrumente
 	}, nil
 }
 
+// NewRawMetaFetcher returns basic meta fetcher without proper handling for eventual consistent backends or partial uploads.
+// NOTE: Not suitable to use in production.
+func NewRawMetaFetcher(logger log.Logger, bkt objstore.InstrumentedBucketReader) (*MetaFetcher, error) {
```

Comment:
I think it is not a good practice to have a public method that is only for testing purposes? I only find this method used in unit tests, then is it better to move this method to that package?

Reply:
Do you have any suggestion, if I want to reuse this across tests?